packages/components/src/components/StatusIcon/StatusIcon.stories.js

```diff
@@ -34,12 +34,18 @@ export default {
   title: 'Components/StatusIcon'
 };
 
-export const Queued = args => <StatusIcon {...args} />;
+export const Cancelled = args => (
+  <StatusIcon reason="TaskRunCancelled" status="False" {...args} />
```

Comment:
Should we add a `<StatusIcon reason="Cancelled" status="False" {...args} />` story as well?

Reply:
Yep, good point. I've added both the legacy `PipelineRunCancelled` and the upcoming `Cancelled`